components/PostDetails/index.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from "react";
+import { documentToReactComponents } from "@contentful/rich-text-react-renderer";
+import Image from "next/image";
+import Link from "next/link";
+import { IPostProps } from "@components/Post";
+import styles from "./PostDetails.module.css";
+
+interface IPostDetailsProps {
+  post: IPostProps;
+}
+
+const PostDetails: React.FC<IPostDetailsProps> = ({ post }) => {
+  return (
+    <React.Fragment>
```

Comment:
Prefer the shorthand syntax:

```suggestion
    <>
```